lib/msal-node/src/cache/CacheManager.ts

```diff
@@ -5,10 +5,10 @@
 
 import { Storage } from './Storage';
 import {
-    ClientAuthError, StringDict,
+    ClientAuthError,
     StringUtils
 } from '@azure/msal-common';
-import { InMemoryCache, JsonCache } from "cache/serializer/SerializerTypes";
+import { InMemoryCache, JsonCache, Dict } from "cache/serializer/SerializerTypes";
```

Comment:
I'm surprised this works, I would expect this to need a leading `./`: `./cache/serializer/SerializerTypes`

Reply:
This will throw in the tests, and possibly the build - for some reason the vscode compiler doesn't catch it though

Reply:
Addressed.